package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "bimedia",
+  "version": "1.0.0",
+  "description": "",
+  "dependencies": {
+    "http-server": "^0.9.0"
```

Comment:
voilà le master que j'ai ouvert hier soir, juste avec Firefox 

----- Mail original -----

De: "Morgan JARRY" <notifications@github.com> 
À: "dilache/bimedia" <bimedia@noreply.github.com> 
Cc: "dilache" <dilache@laposte.net> 
Envoyé: Vendredi 26 Février 2016 09:14:57 
Objet: Re: [bimedia] Initialisation du projet (#2) 



In package.json : 
> @@ -0,0 +1,22 @@
> +{
> +  "name": "bimedia",
> +  "version": "1.0.0",
> +  "description": "",
> +  "dependencies": {
> +    "http-server": "^0.9.0" 


Cette dépendance est importante, elle permet d'installer http-server qui est utilisé par run.sh pour lancer le serveur afin de naviguer sur ton site. 

— 
Reply to this email directly or view it on GitHub .